spec/models/category_spec.rb

```diff
@@ -539,6 +539,7 @@
       @category.destroy
       expect(Category.exists?(id: @category_id)).to be false
       expect(Topic.exists?(id: @topic_id)).to be false
+      expect(Topic.with_deleted.exists?(id: @topic_id)).to be true
```

Comment:
We can remove the assertion above as it is testing the same thing.

Reply:
I merged them into one line :)